addons/TweenSuite/TweenAnimation.gd

```diff
@@ -167,6 +168,8 @@ class IntervalTweenerAnimator extends TweenerAnimator:
 		return "Interval Tweener"
 	
 	func apply_to_tween(tween: Tween, root: Node):
+if randomize_time and time > 0:
+			time = randf_range(0, time)
```

Comment:
This will modify the time stored in the animator and it's going to be wrong on subsequent animations.